content/docs/guides/scim/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: SCIM 2.0 Integration
+meta_desc: This page provides an overview of how to configure any SAML 2.0 identity provider
+           with the Pulumi Console.
+menu:
+    userguides:
+        weight: 9
+        identifier: scim
+
+---
```

Comment:
From an organization perspective, would it make sense to combine this page with the existing https://www.pulumi.com/docs/guides/saml/ page? Feels like the actions are close enough together that we could perhaps have a combined SAML/SCIM User Guides page.

Reply:
Yeah, we probably could; however, there would be some additional cascading work to align and generalize the content on the SAML page, changing link titles, etc. I wasn't sure if we wanted to hold up this release to do that work or if it could come later.

Reply:
Nah, let's do it later then.